src/common/com/intellij/plugins/haxe/model/HaxeAbstractEnumModel.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2018 Ilya Malanin
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intellij.plugins.haxe.model;
+
+import com.intellij.plugins.haxe.lang.psi.HaxeAbstractBody;
+import com.intellij.plugins.haxe.lang.psi.HaxeAbstractClassDeclaration;
+import com.intellij.plugins.haxe.lang.psi.HaxeVarDeclaration;
+import com.intellij.plugins.haxe.util.HaxeAbstractEnumUtil;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class HaxeAbstractEnumModel extends HaxeAbstractClassModel implements IHaxeEnumModel {
+  public HaxeAbstractEnumModel(@NotNull HaxeAbstractClassDeclaration haxeClass) {
+    super(haxeClass);
+  }
+
+  @Override
+  public HaxeEnumValueModel getValue(@NotNull String name) {
+    HaxeVarDeclaration value = getValueDeclarationsStream()
+      .filter(declaration -> name.equals(declaration.getName()))
```

Comment:
I don't understand why you decided that `equals` is case insensitive. Can you explain please

Reply:
Hah.  I didn't speak clearly.  I meant to ask "Would you prefer if the name search was case insensitive?"  I was wondering about the use case and whether it could be that a developer was typing in the wrong case and we would want to supply values for, say, completions or something else.  I'm just not clear on the use case.